packages/_server/src/config/cspellConfig.ts

```diff
@@ -491,6 +491,7 @@ interface CSpellSettingsPackageProperties extends CSpellSettings {
      *       "cpp",
      *       "csharp",
      *       "css",
+     *       "dart",
```

Comment:
This line isn't needed. I would like to avoid adding more file types to this list.

This line will enable it instead:
https://github.com/streetsidesoftware/cspell-dicts/blob/ab609b1789c46d3b19699f9468e4b29380652d34/dictionaries/dart/cspell-ext.json#L15

Reply:
Removed!